docs/api-reference/contexts.md

```diff
@@ -158,20 +158,49 @@ The [Koa Context](https://github.com/koajs/koa/blob/master/docs/api/context.md)
 invoke<T>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>
 ```
 
-Used to invoke workflows, transactions, and communicators.
-The syntax for invoking function `foo` in class `Bar` with argument `baz` is:
+Invoke a transaction or communicator.
+To invoke workflows, use [`invokeWorkflow`](#handlerctxtinvokeworkflow) or [`startWorkflow`](#handlerctxtstartworkflow) instead.
+The syntax for invoking function `fn` in class `Cls` with argument `arg` is:
 
 ```typescript
-handlerCtxt.invoke(Bar).foo(baz)
+const output = await handlerCtxt.invoke(Cls).fn(arg)
 ```
 
-When calling transactions or communicators, `invoke()` asynchronously returns the function's output.
-When calling workflows, `invoke()` asynchronously returns a [`handle`](./workflow-handles) for the workflow.
-
-Note that the DBOS runtime will supply the context to invoked functions.
+You don't need to supply a context to the invoked function&#8212;the DBOS Transact runtime does this for you.
 You can optionally provide a UUID idempotency key to the invoked function.
 For more information, see our [idempotency tutorial](../tutorials/idempotency-tutorial.md).
 
+#### `handlerCtxt.invokeWorkflow`
+
+```typescript
+invokeWorkflow<T>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>
+```
+
+Invoke a workflow and wait for it to complete, returning its result.
```

Comment:
I know it's right below here but I'd mention start.  (If you do not want to wait for the workflow to complete, see startWorkflow below.)